src/api/users/users.services.ts

```diff
@@ -82,31 +82,35 @@ export const createUser = (input: any) => {
 }
 
 // update user
-export const updateUser = (id: string, input: any) => {
-  const { 
+export const updateUser = (id: string | undefined, input: any) => {
```

Comment:
why could be this id undefined?

Reply:
I added it because it was giving me a mistake somewhere else, saying that id could be undefined so I added it here to solve that

Reply:
ok, this could require a check and a possible fix in next PR, no problem.